src/Microsoft.Xaml.Behaviors.DesignTools/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="NuGet.Build.Tasks.Pack" version="5.0.0" targetFramework="net461" developmentDependency="true" />
```

Comment:
You added the NuGet.Build.Tasks.Pack NuGet package to Microsoft.Xaml.Behaviors.DesignTools, but the pack target itself is in Microsoft.Xaml.Behaviors so this NuGet package should be too.